build/automation/var/profile/local.mk

```diff
@@ -36,7 +36,7 @@ COGNITO_USER_POOL_CLIENT_ID := testUserPoolClientId
 COGNITO_USER_POOL_CLIENT_SECRET := testUserPoolClientSecret
 COGNITO_USER_POOL_ID := testUserPoolId
 
-POSTCODE_MAPPING_SERVICE_URL := https://uec-dos-api-pc-dev-uec-dos-api-pc-ingress.k8s-nonprod.texasplatform.uk/api
+POSTCODE_MAPPING_SERVICE_URL := http://mockservice.sfs.test:8080/api
```

Comment:
Is it possible to rename this URL so it's clearer what it is. Something like: mock.postcodeapi.sfs:8080/api ?